deploy/roles/sas/tasks/main.yml

```diff
@@ -1,9 +1,11 @@
 ---
 - name: Install yum dependencies
   yum:
-    name: libXext.i686
+    name: "{{ item }}"
     state: present
-  when: install_sas
+  with_items:
+    - libXext.i686
```

Comment:
@stirlingv We should take another look at why this is necessary, as it doesn't make sense to install an i686 package on an x86_64 CPU...

Reply:
@tanderegg I think you're right. I removed dependency for .i686 library